tests/postgres_tests/test_json.py

```diff
@@ -309,3 +309,18 @@ class CustomJSONField(forms.JSONField):
 
         field = CustomJSONField()
         self.assertIsInstance(field.widget, widgets.Input)
+
+    def test_already_converted_value(self):
+        field = forms.JSONField()
+        vals = [
+            ['a', 'b', 'c'],
+            {'a': 1, 'b': 2},
+            1,
+            True,
+            None,
+        ]
+        for val in vals:
+            try:
+                self.assertEqual(field.clean(val), val)
+            except ValueError:
```

Comment:
Yes, we don't need a try/except/fail. Hiding the original exception makes debugging more difficult and I've removed this pattern wherever I found it in Django's test suite.

Please see the build failure for the JSON test failure and uncheck "Patch needs improvement" on the ticket when things are passing.